### PR TITLE
Add payment request for pktchat_upgrade Milestone 1 

### PR DESCRIPTION
**Pull request by thierrymarianne**
Provided that the maximum requested PKT contribution was originally set to 275K PKT for this project, payment calculation for Milestone 1 reveals that without significantly reducing (by about a factor 1/2) the total requested contribution, the numbers won't add up at current PKT_USDT trading price and EUR/USD change rate. 

I'm very happy the project got accepted and I'm willing to go forward under condition that we'll spread this recalculation to the other milestones so that the project remains viable for both the network steward and myself. 

To be more specific, I think we can not afford [Milestone 10](https://github.com/pkt-cash/ns-projects/blob/master/projects/2022_02_20-pkt.chat_version_upgrade.md#milestones-10-optional) anymore, which was deemed optional from the start (In the future perhaps, I'll submit another proposal to the ns for additional support if need be).

Depending on how the kick-off goes, I'll make the necessary adjustments so that both max contribution in PKT, contribution per milestone and requested amount in national currency are consistent again and reflected accordingly in the proposal.

**In a nutshell, let us schedule the kick-off 💯 and have this chat 🗣️ upgrade done 🎯**

I'm looking forward to hearing you on this.

### CODE REVIEW

**Comment by thierrymarianne** [comment]
pkt-price

[USDT/PKT trading price on the 9th of April 2022](https://global.bittrex.com/Market/Index?MarketName=USDT-PKT) available from bittrex and USD / EUR exchange rate from [finance.yahoo.com](https://finance.yahoo.com/quote/USDEUR=X/) on the same day have been used to calculate Milestone 1 requested kickoff payment amount.

**Comment by cjdelisle** [comment]
I'm going to merge this because it seems uncontroversial.
